packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerSubItemAnimatedExpandableContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useExpandedAnimation } from '@/settings/hooks/useExpandedAnimation';
+import styled from '@emotion/styled';
+import { AnimatePresence, motion } from 'framer-motion';
+import { ReactNode } from 'react';
+
+const StyledAnimatedContainer = styled(motion.div)`
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+`;
+
+type NavigationDrawerSubItemAnimatedExpandableContainerProps = {
+  children: ReactNode;
+  isOpen?: boolean;
+};
+
+export const NavigationDrawerSubItemAnimatedExpandableContainer = ({
```

Comment:
I wonder if that should be a common component that could be used in existing places? Looks really basic, not linked to `NavigationDrawerSubItem` specifically?

Reply:
Makes sense! We have `AdvancedSettingsWrapper` and `ExpandableContainer`. They both are specific to their names. Instead we could have a simple common component `AnimatedExpandableContainer` (naming could be better). Introduce transition constants for specific needs, for eg we have
```
export const ADVANCED_SETTINGS_ANIMATION_DURATION = {
  opacity: 0.2,
  size: 0.4,
};
```
we could have `NAVIGATION_DRAWER_SUB_ITEM_ANIMATION_DURATION` and have it as 300ms?